plugins/modules/inventory_intent.py

```diff
@@ -2449,8 +2520,8 @@ def get_diff_merged(self, config):
                     if response.get('role') == device_role_args.get('role'):
                         self.status = "success"
                         self.result['changed'] = False
-                        log_msg = "Device Role - {0} same in Cisco DNA Center as well, no updation needed".format(device_role_args.get('role'))
-                        self.log(log_msg)
+                        log_msg = "Device Role '{0}' same in Cisco Catalyst Center as well, no updation needed".format(device_role_args.get('role'))
```

Comment:
"The device role '{0}' is already set in Cisco Catalyst Center, no update is needed.".format(device_role_args.get('role'))

Reply:
added.